Cargo.toml

```diff
@@ -99,11 +94,9 @@ http = "1.3.1"
 humantime = "2.1.0"
 itertools = "0.12.1"
 k256 = "0.13.4"
-lazy_static = "1.5.0"
```

Comment:
Oh I thought we got rid of this one already. Perhaps it found it's way back in a merge or something :sweat_smile:

Reply:
I thought the same :)